plugins/plugin-auth/src/local-auth/admin/generators/admin-crud-reset-password-action/templates/routes/-components/password-reset-dialog.tsx

```
@@ -0,0 +1,155 @@
+// @ts-nocheck
```

ESLint

[error] Do not use "@ts-nocheck" because it alters compilation errors.

(@typescript-eslint/ban-ts-comment)